src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,48 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("Set Collection에 대한 학습 테스트")
+public class SetTest {
+	private Set<Integer> numbers = new HashSet<>();
+
+	@BeforeEach
+	void setUp() {
+		numbers = new HashSet<>();
+		numbers.add(1);
+		numbers.add(1);
+		numbers.add(2);
+		numbers.add(3);
+	}
+
+
+	@Test
+	@DisplayName("size() 메소드를 통해 크기를 확인할 수 있다.")
+	void getSize() {
+		assertThat(numbers.size()).isEqualTo(3);
+	}
+
+	@ParameterizedTest
+	@DisplayName("주어진 값을 모두 담고 있다.")
+	@ValueSource(ints = {1, 2, 3})
+	void containsElement(int element) {
+		assertThat(numbers.contains(element)).isTrue();
+	}
```

Comment:
```suggestion
	@Test
	@DisplayName("size() 메소드를 통해 크기를 확인할 수 있다.")
	void getSize() {
		assertThat(numbers).hasSize(3);
	}

	@ParameterizedTest
	@DisplayName("주어진 값을 모두 담고 있다.")
	@ValueSource(ints = {1, 2, 3})
	void containsElement(int element) {
		assertThat(numbers).contains(element);
	}
```
컬렉션을 쉽게 테스트할 수 있도록 메소드를 제공하고 있습니다. 🤩